.evergreen/config.yml

```diff
@@ -69,7 +69,7 @@ functions:
            PROJECT_DIRECTORY: "$PROJECT_DIRECTORY"
            PREPARE_SHELL: |
               set -o errexit
-              set -o xtrace
+              set +o xtrace
```

Comment:
Why the change to `+o xtrace` here rather than removing the existing line as was done in other files?

Reply:
I had originally left this in to disable xtrace when preparing a shell, but any new shell most likely starts out with it disabled anyways. I've removed it 👍